lib/middlewares/project.ts

```diff
@@ -1,21 +1,17 @@
 import { Validator } from '@voiceflow/backend-utils';
 import VError from '@voiceflow/verror';
-import { NextFunction, Request, Response } from 'express';
+import { NextFunction, Response } from 'express';
 
 import { validate } from '@/lib/utils';
 import { CreatorDataApi } from '@/runtime';
+import { Request } from '@/types';
 
 import { AbstractMiddleware } from './utils';
 
-const { param, header } = Validator;
+const { header } = Validator;
 const VALIDATIONS = {
-  PARAMS: {
-    VERSION_ID: param('versionID')
-      .optional()
-      .isString(),
-  },
   HEADERS: {
-    VERSION_ID: header('versionID')
+    VERSION: header('versionID')
```

Comment:
```suggestion
    VERSION_ID: header('versionID')
```

Would have to update references too.